### PR TITLE
Making buttons edit, destroy vidible when you are event host

### PR DESCRIPTION
**Pull request by maciek1995**
#78

### CODE REVIEW

File: app/views/events/index.html.slim [hunk 1]
```diff
@@ -19,6 +19,14 @@ h1 Events
...
+                    - if policy(event).edit?
+                      = link_to edit_event_path(event)
+                        div class="btn text-white"
+                          = 'Edit'
+                    - if policy(event).destroy?
+                      = link_to event, :method => :delete, :data => {:confirm => 'Are you sure?'}
+                        div class="btn text-white"
+                          = 'Destroy'
 .container
   -if current_user
```
Comment (Snieezy) [inline app/views/events/index.html.slim]: missing space

File: app/views/events/show.html.slim [hunk 2]
```diff
@@ -19,10 +19,14 @@ p#notice= notice
       .text-field = @event.end_t.strftime("%d/%m/%Y %H:%M")
 .container.padding
   .event-buttons
-    -if current_user
+    - if policy(@event).edit?
       = link_to edit_event_path(@event)
         div class="btn text-white"
           = 'Edit'
+    - if policy(@event).destroy?
+      = link_to @event, :method => :delete, :data => {:confirm => 'Are you sure?'}
```
Comment (Snieezy) [inline app/views/events/show.html.slim]: gotar mentioned to use new hash convention:
use method: :delete instead of :method => :delete

**Comment by Snieezy** [comment]
+1

**Comment by gotar** [comment]
ok you can merge

**Pull request merged**